dcm4che-core/src/main/java/org/dcm4che3/data/Fragments.java

```diff
@@ -87,6 +87,10 @@
     private final VR vr;
     private final boolean bigEndian;
 
+    public Fragments(VR vr, boolean bigEndian, int initialCapacity) {
```

Comment:
This is just a change to make it look more like the master branch because I was testing this it this way, not otherwise used yet, but I plan to make more commits over time to improve agreement between master and the GC branch